internal/display/display.go

```diff
@@ -0,0 +1,46 @@
+package display
+
+import (
+	"errors"
+	"io"
+	"os"
+	"os/exec"
+)
+
+func Display(defaultOutput, errorOutput io.Writer, contents []byte, page bool) error {
```

Comment:
could you add some tests for this?

Reply:
i also want us to walk through how the default works on windows for paging. It'd be worth looking at what git itself does here.

Reply:
I have added some tests. I work on a Mac, so I don't think I can check if paging works differently for Windows.

Reply:
Gotcha. I think we might need a _linux and _windows version of this file, basically. @wlynch wdyt?

Reply:
You can probably get away with using [exec.LookPath](https://pkg.go.dev/os/exec#LookPath)

For windows support, it looks like `more` is a supported command so if you iterate though

1. $PAGER
2. less
3. more
4. stdout

You'll hit something that will be supported.